urls.js

```diff
@@ -37,7 +37,12 @@ module.exports = {
     },
 
     representative: {
-        representative:                 '/representative'
+        hasRepresentative:              '/representative',
```

Comment:
`hasRepresentative ` and `/representative` in not consistent with the naming convention throughout this file.